File: libs/widgets/MultiPowUpload/public_html/FileProcessingScripts/RedirectUploadScripts/PERL/result.pl
#!/usr/bin/perl -w 

use CGI; 

$query = new CGI; 

print $query->header ( ); 
print $query->start_html();

@names = $query->param;
%hnames = ();
foreach $item (@names) { $hnames{$item} = $query->param($item) }
for($i=0; $i<@names; $i++)
{
	if(defined($hnames{"MultiPowUploadFileName_$i"}) and defined($hnames{"MultiPowUploadFileSize_$i"})) 
	{
		print "File with name ", $hnames{"MultiPowUploadFileName_$i"},
			" and size ", $hnames{"MultiPowUploadFileSize_$i"}, " uploaded successfully",
			$query->br();
	}

	if(defined($hnames{"MultiPowUploadUnuploadedFileName_$i"})) 
	{
		print "File with name ", $hnames{"MultiPowUploadUnuploadedFileName_$i"},
			" and size ", $hnames{"MultiPowUploadUnuploadedFileSize_$i"}, " was not uploaded.",
			$query->br();
	}

}
print $query->end_html();

File: libs/widgets/MultiPowUpload/public_html/FileProcessingScripts/RedirectUploadScripts/PHP/result.php
<?php

for($i=0; $i<count($_POST); $i++)
{
	if(isset($_POST["MultiPowUploadFileName_". $i]))
	{
		echo "File with name " . $_POST["MultiPowUploadFileName_". $i]. " and size " . $_POST["MultiPowUploadFileSize_" . $i] . " uploaded successfully<br>";
	}
}


for($i=0; $i<count($_POST); $i++)
{
	if(isset($_POST["MultiPowUploadUnuploadedFileName_". $i]))
	{
		echo "File with name " . $_POST["MultiPowUploadUnuploadedFileName_". $i]." and size " . $_POST["MultiPowUploadUnuploadedFileSize_" . $i] ." was not uploaded<br>";
	}
}

?>